README.md

```diff
@@ -213,6 +212,78 @@ fastify.listen(3000, err => {
 })
 ```
 
+#### Example using cookie
+
+Storing JWT in Cookie mean that your app maybe vunerable to XSS attack and must be protected with CSRF token,
+consider that as a best practice. but storing JWT on cookies makes your REST API arent Stateless anymore, choose what fit for you.
+You can use [crsf](https://www.npmjs.com/package/csrf) or other library that may suit your need.
+
+```js
+const { readFileSync } = require('fs')
+const path = require('path')
+const fastify = require('fastify')()
+const jwt = require('fastify-jwt')
+const Redis = require('ioredis')
+
+const redis = new Redis()
```

Comment:
```suggestion
// docker run -p 6379:6379 --name redis-test redis
const redis = new Redis({ port: 6379, host: '127.0.0.1' })
```

Reply:
actualy, just `new Redis()` would be set to default port but this more informative.